deepmd/pt/utils/env_mat_stat.py

```diff
@@ -147,6 +148,9 @@ def iter(
                 self.descriptor.get_nsel(),
                 self.last_dim,
             )
+            exclude_mask = self.descriptor.get_emask(nlist, extended_atype).view(
+                coord.shape[0] * coord.shape[1], -1
+            )
```

Comment:
we do not expect the descriptors supporting excluding types. manipulating the nlist instead. see https://github.com/deepmodeling/deepmd-kit/blob/93e253eb04085a96d8451197a9b2314d34033f13/deepmd/pt/model/atomic_model/base_atomic_model.py#L98-L101